securedrop/models.py

```diff
@@ -31,6 +32,9 @@
 # precisely control which code paths are exercised.
 if os.environ.get('SECUREDROP_ENV') == 'test':
     LOGIN_HARDENING = False
+    ARGON2_PARAMS = dict(memory_cost=2**3, rounds=1, parallelism=1)
```

Comment:
hey is this strictly necessary (i.e. did this provide a significant speedup)? I'm asking because having test-only code paths in prod code is in my opinion an anti-pattern and this if/else in particular is one that we can hopefully snip out some point soon

Reply:
```
heartsucker@7bee4bf4317f:/home/heartsucker/code/freedomofpress/securedrop/securedrop$ python -m timeit 'from passlib.hash import argon2; argon2.using(memory_cost=2**3, rounds=1, parallelism=1).hash("test")'
10000 loops, best of 3: 189 usec per loop
heartsucker@7bee4bf4317f:/home/heartsucker/code/freedomofpress/securedrop/securedrop$ python -m timeit 'from passlib.hash import argon2; argon2.using(memory_cost=2**16, rounds=4, parallelism=1).hash("test")'
10 loops, best of 3: 284 msec per loop
```

So I'm gonna say no, not necessary.

Reply:
This was removed.